analysis_driver/util/bash_commands.py

```diff
@@ -124,6 +124,12 @@ def bamtools_stats(bam_file, output_file):
     app_logger.debug('Writing: ' + cmd)
     return cmd
 
+def samtools_stats(bam_file, output_file):
+    samtools_bin = cfg.query('tools', 'samtools')
+    cmd = '%s stats %s > %s' % (samtools_bin, bam_file, output_file)
+    app_logger.debug('Writing: ' + cmd)
+    return cmd
+
```

Comment:
Are we wanting to keep `bamtools_stats` above? Looks like it's been removed from the rest of the pipeline, including the tests